documentation/publish/promoting-your-extension.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: sidebar
+title: Promoting your extension or theme
+permalink: /documentation/publish/promoting-your-extension/
+published: false
+topic: Publish
+tags: [add-on, distribution, guide, promote]
+contributors: [irenesmith, rebloor]
+last_updated_by: irenesmith
+date: 2019-03-18 05:01:28
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Promoting your extension or theme
+
+So you’ve written a great extension, but only a few people have found and installed it from [addons.mozilla.org](https://addons.mozilla.org) (AMO)—despite your efforts to create an appealing listing. So, how do you encourage more people to install your extension? This page looks at how.
```

Comment:
missing link for "create an appealing listing"

Reply:
Link added